src/main/java/pt/cryptosaft/demo/FastSaftEncrypt.java

```diff
@@ -137,8 +137,11 @@ private static IterationParameters processEvent(XMLStreamReader xmlr, Reader rea
 					iParam.getTree().add(xmlr.getLocalName());
 
 					//System.out.println(iParam.getCurrentBranch());
-					iParam.setElementToCipher(elementsToCypher.contains(iParam.getCurrentBranch()));
-					iParam.setElementToCipher(true);
+				        
+					if (elementsToCypher.contains(iParam.getCurrentBranch())) {
```

Comment:
Thx! Can you please declare all the proper elements in a constant?
https://cryptosaft.assoft.org/informacao-tecnica/cryptosaf-t/elementos